.github/workflows/ci_cd.yml

```diff
@@ -0,0 +1,76 @@
+name: CI/CD
+
+on:
+  push:
+    branches: 
+    - master
+  pull_request:
+    branches: 
+    - master
+
+jobs:
+  test:
+    name: Test on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version:
+        - 10
+        - 12
+        os: 
+        - ubuntu-latest
+        - windows-latest
+        - macOS-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setting up Node.js ${{ matrix.node_version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node_version }}
+
+    - name: Get npm cache directory
+      id: npm-cache
+      run: |
+        echo "::set-output name=dir::$(npm config get cache)"
+    - uses: actions/cache@v1
+      with:
+        path: ${{ steps.npm-cache.outputs.dir }}
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+          
+    - name: Installing npm modules
+      run: npm install
+
+    - name: Running Tests
+      run: npm run test
+
+    - name: Electron Builder Action
+      uses: samuelmeuli/action-electron-builder@v1.3.0
+      with:
+        github_token: ${{ secrets.github_token }}
+        mac_certs: ${{ secrets.mac_certs }}
+        mac_certs_password: ${{ secrets.mac_certs_password }}
+
+    - name: See files in dist folder
+      run: ls ./dist
```

Comment:
Here is one more thing you can do to make the artifacts convenient. Rename the output directory to have the same name as it is on GitHub releases. For example, when building for macOS, the electron builder may spit out a dmg named `Zulip-macOS-4.0.3.dmg` when on releases page it is `Zulip-4.0.3.dmg`.  Let me know if it unclear.

Reply:
I think these things are done by modifying package.json. I explored the way in which GitHub action can access package.json file content, but can't find any way to do this. Currently we get  ubuntu-latest > Zulip-4.1.0-beta-amd64.deb this type of name.
Also, we can name the artifact folder according to the version of Zulip by adding one more environment variable.

Reply:
For this case, just using a node script that renames the file is enough but we can revisit this later.